src/main/java/slimeknights/mantle/command/BookTestCommand.java

```diff
@@ -1,74 +0,0 @@
-package slimeknights.mantle.command;
-
-import com.mojang.brigadier.builder.LiteralArgumentBuilder;
-import com.mojang.brigadier.context.CommandContext;
-import com.mojang.brigadier.exceptions.CommandSyntaxException;
-import net.minecraft.commands.CommandSourceStack;
-import net.minecraft.commands.Commands;
-import net.minecraft.commands.arguments.ResourceLocationArgument;
-import net.minecraft.resources.ResourceLocation;
-import net.minecraft.server.level.ServerPlayer;
-import slimeknights.mantle.Mantle;
-import slimeknights.mantle.network.MantleNetwork;
-import slimeknights.mantle.network.packet.OpenNamedBookPacket;
-
-import java.util.HashSet;
-import java.util.Set;
-
-/** Command that opens a test book */
-public class BookTestCommand {
-  private static final Set<ResourceLocation> BOOK_SUGGESTIONS = new HashSet<>();
-
-  /**
-   * Registers this sub command with the root command
-   * @param subCommand  Command builder
-   */
-  public static void register(LiteralArgumentBuilder<CommandSourceStack> subCommand) {
-    subCommand.requires(source -> source.hasPermission(MantleCommand.PERMISSION_GAME_COMMANDS) && source.getEntity() instanceof ServerPlayer)
-      .then(Commands.argument("id", ResourceLocationArgument.id()).suggests(MantleCommand.REGISTERED_BOOKS)
-        .executes(BookTestCommand::runBook))
-      .executes(BookTestCommand::run);
-  }
-
-  /**
-   * Adds a book suggestion to the list of suggestions
-   * @param suggestion The suggestion to be added
-   */
-  public static void addBookSuggestion(ResourceLocation suggestion) {
```

Comment:
If we end up back porting this to 1.19.2, we will need to keep this method around. Can make it just a deprecated dummy method that does nothing, just don't want to require a tinkers update for this change.